department-of-veterans-affairs/vets-website
# Revert asset hashes removal

## Description
We discovered that some assets weren't receiving the correct `Cache-Control` header, which was causing some issues. After some investigation, it was unclear why this was happening, so we decided to revert the commit that removed the hashes from the asset filenames for cache busting.

Reverts https://github.com/department-of-veterans-affairs/vets-website/pull/12249 and ensures that we can still run `yarn build:content` without needing Webpack assets first.

## Testing done
Ensured it was working for a localhost build
1. `rm -r build/localhost/`
1. `yarn build:content`
1. `yarn watch`
1. Navigate to `localhost:3001/find-locations/`
1. See that the application loads with the non-hashed assets

Ensured it was working for a prod build
1. `rm -r build/vagovprod/`
1. `NODE_ENV=production yarn build --buildtype vagovprod`
1. `http-serve build/vagovprod/ -p 3001`
1. Navigate to `localhost:3001/find-locations/`
1. See that the application loads with the hashes in the asset names

## Screenshots
N/A

## Acceptance criteria
- [x] Localhost builds are unaffected
- [x] Prod builds use hashes in the asset names to bust the cache
